cv_next/app/feed/components/CVItem.tsx

```diff
@@ -24,9 +24,8 @@ export default function CVItem({ cv }: CVCardProps) {
     Definitions.PLAICEHOLDER_IMAGE_DATA
   );
 
-  const categoryLink = `/feed?category=${Categories.category[
-    cv.category_id
-  ].toLowerCase()}`;
+  const generateCategoryLink = (categoryNumber: number) =>
```

Comment:
This can be exported to a util file, right? I think it'd be better (and more efficient react-wise)